ports/srpc/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO sogou/srpc
+    REF v0.9.3
+    SHA512  e64cfec279f833ad24b1942ef2b572fc4bd3bfc5f9d623cd5d25b3c869d2ff9b35250cc814e8aaeb919ebed0929c44407eb9abb2e6793cfdc967708210f5f7e3
+    HEAD_REF master
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
```

```suggestion
vcpkg_cmake_configure(
    SOURCE_PATH ${SOURCE_PATH}
```
We prefer to use new function instead of the deprecated one for new ports.

Please see https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_configure_cmake.md.